### ISSUE TITLE
Exit VMWare Console via Remmina in 1.2.0

### ISSUE THREAD

**Issue by flickerfly** [FreeRDP/Remmina#522]
I've recently upgraded Remmina to 1.2.0 from 0.99.0. (I think, otherwise whatever Ubuntu 14.04 has in the default repos) I see lots of great improvement, but am struggling with the handling of keyboard and mouse actions to the remote computer. 

I first noticed this when opening a VMWare Console on a remote computer. I was unable to use Ctrl+Alt to exit the console which means I lost full control of the computer since (if you don't have VMWare tools installed) it locks all input to the console. Being unable to exit from the console creates the problem of how to regain without gaining access to the remote computer by other means.

I've seen similar issues with key combos I've gotten around via other means, but that one is a real sticky since I can't find a work-around for systems without vmware tools installed.

I've also noticed that, when I have the remote resolution set to larger that the Remmina window, scolling of the mouse no longer scrolls the remmina window up and down to get to other parts of the screen. I used to use that a lot to basically hide the start bar at the bottom of a remote windows session until I had need of it. This is of course less significant.

Am I missing somethings about how this should be handled, are these known issues that are simply the new way or am I dealing with a bug? 

Thanks!

**Comment by giox069** [comment]
For VMWare vSphere Client: are you pressing the left or the right "Ctrl+Alt" ? Please note that in remmina the right Ctrl key is the default "Host key", and is used for special purposes (ie: host+F=fullscreen). I'm using Ubuntu 14.10 with remmina-next PPA to access many Windows 7 with ESXi console (vSphere Client) installed, and I can escape from the vsphere client screen capture with no problems by pressing the left "Ctrl+Alt" keys.
The host key can be changed in the global Remmina preference dialog if you don't like it.

Please also try to enable "keyboard events grab" by pressing the keyboard icon on the remmina toolbar when connected: keyboard grabbing should also help with other key combos.

For "window not scrolling" in fullscreen mode when the remote screen is larger than local desktop: when in fullscreen mode, go on the floating toolbar at the top of the screen and click on the down arrow (3rd icon from the left). A menu will popup, please choose "Viewport fullscreen mode" there.
Unfortunately there should be a bug: sometimes Xorg or the window manager will be unable to put the window in fullscreen mode. Just disconnect and reconnect to enter in fullscreen mode.

I don't really know if you are dealing with a bug or something different, but I thing these questions areu are very useful to build a new FAQ list in the remmina Wiki :)

**Comment by flickerfly** [comment]
Great help, thank you. I figured I couldn't be the only one. I'd moved that hotkey to the Left Ctrl and I was using the left Ctrl+Alt. I changed the hotkey back to the right an things seem to have solved my problem. I guess I didn't think it would steal all left Ctrl actions, just the ones it used. I avoid the right Ctrl cause half the time it is all the way over on my mouse for Windows use and it is just SOOO much work to move my lazy hand over to the keyboard. Maybe my Caps Lock ... 

My scrolling issue isn't in full screen mode, but in a maximized tabbed window without using the scaling feature. Maybe I should just use full screen mode or scaled mode anyway. 

Glad to help build the FAQ :-)
